### ISSUE TITLE
Integer values can't be negative

### ISSUE THREAD

**Issue by blondie63** [FlutterFlow/flutterflow-issues#551]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.

### Current Behavior

In many part of FF you cannot use negative integer value !
But for example i can create an App State variable type integer and default value -1

This is out of standard ! why ??

### Expected Behavior

I'm expecting that FF use standard types

### Steps To Reproduce

Create a page
Add a list view
Set a backend query as API Call
Edit API Call and add a variable with type Integer and try to set default to -1, you can 't ! :(

### Bug Report Code

IT4wlsmAsI1Nobhe+KqJb/pvoCMvIjo2aJ0rk8IbGSs8GqLuPe0UWMzBUBFUOP3nY3xIIVn88HMX/MrKjc7AV8A7AzasfJxZzroAWw3McFOVVJOYEYS3b39tHdFbGGq94baVkQhSJrdfSCAtw2abNeqWYAnfJ7qxOUg7HuaoXdeK2SrDX1iXc2URm05KZDPz

### Context

Always

### Environment

```markdown
- FlutterFlow version: 3.1
- Platform:MacOS
- Browser name and version: Chrome 110.0.5481.100
- Operating system and version affected: FF Testing on web
```

**Comment by organicnz** [comment]
Hi @blondie63 . Thank you Mauro for adding this issue 🙂

Confirmed. I could reproduce this issue :)

![Uploading Screenshot 2023-02-28 at 11.01.31 PM.png…]()

**Comment by bulgariamitko** [comment]
I made some tests and it is actually working fine for me. 


https://user-images.githubusercontent.com/7221384/222155275-1a9cb479-8b2a-4f04-84f9-263a43a0ebed.mp4

**Comment by blondie63** [comment]
I still have same problems with last version!

**Comment by bulgariamitko** [comment]
In order to keep our issue tracker useful and maintainable for all users, please follow the bug report template while reporting bugs. We will be happy to re-assess when the necessary information is provided.
 
 Thank you!

**Comment by bulgariamitko** [comment]
I am re-opening this issue because both my video and Tarlan ss are not relevent to the issue the user is posted about. The user ment this issue. This is inside API calls - Variables - Type Integer - Default Value -1

![whats-new-FlutterFlow](https://user-images.githubusercontent.com/7221384/223394261-f0d88a35-f6c3-438c-9a00-7ae921de4f8c.png)

**Comment by agreaves** [comment]
Thanks @blondie63 for the bug report! This will be fixed in our next deploy.

**Comment by blondie63** [comment]
fixed and released ? version ?

**Comment by agreaves** [comment]
No not yet, on our next deploy (early this coming week).

**Comment by blondie63** [comment]
> No not yet, on our next deploy (early this coming week).

ok, i hope you fix also #589 that it's a big bug  :(

**Comment by blondie63** [comment]
> No not yet, on our next deploy (early this coming week).

Yesterday March 13th you announced the new version that had actually already been released a few days before but the worst thing is that you didn't insert the fixes for the bugs reported by me and that you closed here on Gitbub as fixed. I honestly don't understand this absurd behavior, do you want your customers to leave FlutterFlow ?
